bindings/kotlin/build.gradle

```diff
@@ -0,0 +1,23 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+apply plugin: 'io.github.gradle-nexus.publish-plugin'
+apply from: "${rootDir}/scripts/publish-root.gradle"
+
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath BuildPlugins.android
+        classpath BuildPlugins.kotlin
+        classpath BuildPlugins.nexusPublish
+    }
+}
+
+
+task clean(type: Delete) {
+    delete rootProject.buildDir
+}
```

Comment:
Is this file generated? If not, should have final newline lke everything else.

Reply:
Same in most / all other `.gradle`, `.kt`, `.kts` files. Is this because of some IDE setting or actually preferred in the Kotlin ecosystem?

Reply:
I honestly have no clue. Didn't touch any of these files post import, but can do.